fractions/microprofile/microprofile-jwt/src/main/java/org/wildfly/swarm/microprofile/jwtauth/MicroProfileJWTAuthFraction.java

```diff
@@ -62,6 +62,20 @@
     @Configurable("thorntail.microprofile.jwt.default-missing-method-permissions-deny-access")
     private Defaultable<Boolean> defaultMissingMethodPermissionsDenyAccess = bool(true);
 
+    @AttributeDocumentation("HTTP header which is expected to contain a JWT token, default value is 'Authorization'")
+    @Configurable("thorntail.microprofile.jwt.token.header")
+    private Defaultable<String> tokenHeader = string("Authorization");
+    @AttributeDocumentation("Cookie name containing a JWT token. This property is ignored unless the 'thorntail.microprofile.jwt.token.header' is set to 'Cookie'")
+    @Configurable("thorntail.microprofile.jwt.token.cookie")
+    private String tokenCookie;
+    @AttributeDocumentation("Default group name. This property can be used to support the JWT tokens without a 'groups' claim.")
+    @Configurable("thorntail.microprofile.jwt.claims.groups")
+    private String defaultGroupsClaim;
+
+    @AttributeDocumentation("JSON path to the claim containing an array of groups. It can be used a token has no 'groups' claim but has the groups set in a different claim")
```

Comment:
It's not "JSON Path". The syntax is actually closer to JSON Pointer, but it's not it either. Avoiding "JSON" + adding an example would be best I think.